tools/debian/cockpit-ws.postinst

```diff
@@ -21,8 +21,8 @@ if [ -d /run/systemd/system ] && [ -n "$2" ]; then
     deb-systemd-invoke try-restart cockpit.service >/dev/null || true
 fi
 
-# set up dynamic motd/issue symlinks on first-time install; don't bring them back on upgrades if admin removed them
-if [ "$1" = "configure" ] && [ -z "$2" ]; then
+# set up dynamic motd/issue symlinks on first-time install or upgrades from < 244 (which moved them out of the .deb)
+if [ "$1" = "configure" ] && dpkg --compare-versions "$2" lt 244; then
```

Comment:
Should not the && be || ?

Reply:
No, && is correct. This should only happen for the "configure" phase, not e.g. for "abort-remove" or "abort-upgrade".

Reply:
But with this change on fresh installs which are *not* upgrades from < 244 this file creation code will not run. 

From what I understand you want to do:
`if is_configure_phase || (is_configure_phase && dpkg --compare-versions "$2" lt 244)`

Reply:
OH I see, after version 244 this file was present. Sorry missed that.

Reply:
`--compare-versions` treats an empty version as earlier than any other, so `'' lt 244` is true, and the code runs as intended. FTR, this isn't always the case, for that there's also `lt-nl` operator ("none is later").